concourse/pipelines/deploy.yml

```diff
@@ -241,6 +241,13 @@ resources:
       location: "Europe/London"
       fire_immediately: true
 
+  - name: every-5-minutes
+    type: cron-resource
+    source:
+      expression: "*/5 * * * 1-5"
```

Comment:
What will Concourse do if the tests start taking longer than 5 minutes?

Reply:
It depends on the job [`serial` value](https://concourse-ci.org/jobs.html#schema.job.serial):

> Default false. If set to true, builds will queue up and execute one-by-one, rather than executing in parallel.

In this case, the `all-smoke-tests` job will run smoke tests in parallel if they start taking longer. This is probably fine for now - if we see we start having build ups of tests, we could increase this to every 10mins (and so on).